genotype/store/trending.py

```diff
@@ -0,0 +1,61 @@
+"""Functions to create trending specific data."""
+import logging
+from genotype.store.models import Analysis, Genotype, Sample
+
+LOG = logging.getLogger(__name__)
+
+
+def build_snp_dict(analysis_id: str) -> dict:
+    """Building a dict of snps for a specific analysis."""
+
+    snp_dict = {}
+    genotypes = Genotype.query.filter(Genotype.analysis_id == analysis_id).all()
+    if not genotypes:
+        LOG.warning('Did not find Genotype data for analysis_id %s', (analysis_id))
+    for genotype in genotypes:
+        snp_dict[genotype.rsnumber] = [genotype.allele_1, genotype.allele_2]
+
+    return snp_dict
+
+
+def compare(analysis_1: dict, analysis_2: dict) -> dict:
```

Comment:
I suggest we name this method something more specific, compare is too general, right?